lib/class-wp-theme-json-gutenberg.php

```diff
@@ -92,7 +92,10 @@ class WP_Theme_JSON_Gutenberg {
 			'palette'        => null,
 		),
 		'custom'     => null,
-		'layout'     => null,
+		'layout'     => array(
```

Comment:
I think later we'll have "type" and a different shape of layout depending of the "type". Maybe it's fine for now to have this but I thought I'd mention in case it's any impactful.

Reply:
When/If we have those new fields we can add them all and/or have some specific logic for `layout` to make sure only the ones that make sense depending on type are allowed.

By porting this to WordPress core we won't allow anything else than what it's shipped in v1 (the two fields in use atm) without the plugin. With the plugin, you'll have those new things. Makes sense?

Reply:
WFM 👍